src/resolvers/resolver.ts

```diff
@@ -19,6 +23,29 @@ const resolvers = {
       if (userExists) throw new Error("Email is taken");
       const newUser = await User.create({ email, password });
       return newUser;
+    },
+    async createUserRole(_: any, { userRole }: any) {
+      const newRole = await UserRole.create({ userRole });
+      return newRole;
+    },
+
+    async updateUserRole(_: any, { id, userRole }: any) {
+          const roleExists = await UserRole.findOne({ userRole: userRole });
```

Comment:
make sure we check the logged in user permission before executing.

For now you can add a TODO for it since we are waitig for the merge of PR #13

Reply:
Feedback applied.